task/06-conditions-n-loops-tasks.js

```diff
@@ -52,7 +52,9 @@ function getFizzBuzz(num) {
  */
```

Comment:
There is not travis-ci link for PR

Reply:
https://travis-ci.org/github/Inquisitor1229/js-assignments/branches